libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -25,12 +27,13 @@ const getMetadata = (el) => [...el.childNodes].reduce((acc, row) => {
 }, {});
 
 class AppPrompt {
-  constructor({ promptPath, id, profileApi } = {}) {
+  constructor({ promptPath, id, profileApi, entName } = {}) {
     this.promptPath = promptPath;
     this.id = id;
     this.profileApi = profileApi;
     this.elements = {};
-
+    // TODO - make sure this is still valid after we have the entitlements mapping
+    this.productName = capitalize(entName);
```

Comment:
If a product name is needed for analytics purposes, we need to go back to a previous requirement, as the entitlement name would not be sufficiently clear. An entitlement name would be something like 'isEntitledAndHasUsedPhotoshop', whereas we just want 'Photoshop'.

We need to add a new field to the section metadata of the document to allow authors to define the product name that should be used here.